### PR TITLE
Allow skipping events that are not reportable

### PR DESCRIPTION
**Pull request by xsuchy**
e.g. lunch break, focus block, deadline reminders

### CODE REVIEW

File: did/stats.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Stats(object):
     option = None
     dest = None
     parent = None
+    skip = None
```
Comment (psss) [inline did/stats.py:25]: This should not be needed. The `skip` option belongs to configuration and it doesn't have to be defined here.

File: did/plugins/google.py [hunk 2]
```diff
@@ -301,6 +304,7 @@ def __init__(self, option, name=None, parent=None, user=None):
             apps = [app.lower() for app in split(config["apps"])]
         except KeyError:
             apps = DEFAULT_APPS
+        self.skip = config.get("skip")
```
Comment (psss) [inline did/plugins/google.py:307]:
```suggestion
        self.skip = config.get("skip", [])
```

To make it working when no `skip` is present.

**Review by psss — CHANGES_REQUESTED** [review]
Thanks for the improvement! Just two suggestions.

**Comment by xsuchy** [comment]
Updated.

**Review by psss — APPROVED** [review]
Thanks for the adjustments, now looks good. Just modified the [commit message](https://did.readthedocs.io/en/latest/contribute.html#commits) for a consistent changelog.

**Pull request merged**